jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/RuntimeDataServiceImpl.java

```diff
@@ -811,6 +811,17 @@ public UserTaskInstanceDesc getTaskById(Long taskId) {
         return null;
 	}
 
+    @Override
+    public UserTaskInstanceDesc getTaskByIdWithSLA(Long taskId) {
+        UserTaskInstanceDesc taskInstanceDesc = getTaskById(taskId);
```

Comment:
I believe this can return `null` so the next line can fail. So we need a null check first.

Reply:
@MarianMacik  ups, sorry too fast, Thanks.
updated.